README.md

```diff
@@ -9,3 +9,6 @@ To run locally using docker:
  - run `docker run --rm -it -v /ABSOLUTE/PATH/TO/PROJECT/:/app composer install` to install dependencies
  - run `docker run --rm -it -v /ABSOLUTE/PATH/TO/PROJECT/:/app composer /app/vendor/bin/phpunit` to execute tests
  - run `docker run --rm -p 80:80 -v /ABSOLUTE/PATH/TO/PROJECT/:/app webdevops/php-nginx:alpine` to run the webserver on port 80
+
+To build a production image:
+- run `docker build --tag kordianbruck/tumcalproxy -f Dockerfile .`
```

Comment:
`-f Dockerfile` is redundant. I don't know if this naming convention is recommended if someone else is building it because the local image will have exactly the same name as the one you uploaded to the public registry. In this case a simple `docker pull kordianbruck/tumcalproxy` would override the local image which might not be desired.

Reply:
ah, I see you already changed it